src/ui/handler/scroll_zoom.js

```diff
@@ -356,6 +376,27 @@ class ScrollZoomHandler {
     reset() {
         this._active = false;
     }
+
+    _addScrollZoomBlocker() {
+        if (this._map && !this._container) {
+            this._container = DOM.create('div', 'mapboxgl-scroll-zoom-blocker-control', this._map._container);
+            const alertMessage = this._map._getUIString('ScrollZoomBlocker.Message');
```

Comment:
I think in this case we should still try to detect the platform and replace `Ctrl` with `⌘` when needed

>  since navigator.platform to determine platform specific responses is not considered best practice, and is in the process of being deprecated

I agree that in general this isn't the best practice, but as long as it always falls back to the `Ctrl` message (and both ctrl and meta always work on both platforms), there shouldn't be much harm in trying to detect it.